mythril/analysis/modules/ether_thief.py

```diff
@@ -22,7 +22,57 @@
 
 """
 
-ARBITRARY_SENDER_ADDRESS = 0xAAAAAAAABBBBBBBBBCCCCCCCDDDDDDDDEEEEEEEE
+
+def _analyze_state(state):
+    instruction = state.get_current_instruction()
+    node = state.node
+
+    if instruction["opcode"] != "CALL":
+        return []
+
+    call_value = state.mstate.stack[-3]
+    target = state.mstate.stack[-2]
+
+    eth_sent_total = BitVecVal(0, 256)
+
+    constraints = node.constraints
```

Comment:
This should Have been copied, after some painful debugging realized that problem was in this file :rofl: